src/ResourceManager/Storage/Commands.Management.Storage/help/Get-AzureRmStorageUsage.md

```diff
@@ -28,14 +28,22 @@ PS C:\>Get-AzureRmStorageUsage
 ```
 
 This command gets the Storage resources usage of the current subscription.
+ 
+
+### Example 2: Get the storage resources usage of specified location
+```
+PS C:\>Get-AzureRmStorageUsage -Location "US West"
```

Comment:
@blueww for the examples in this cmdlet help file, would you mind showing the resulting output that users would see from running the command? You can put the output in the code block with the command.

Reply:
fixed